docs/components/grapesjs.rst

```diff
@@ -0,0 +1,74 @@
+GrapesJS Builder
+################
+
+Currently, Mautic uses the GrapesJS Builder for Emails and Landing pages. 
+
+Setup 
+*****
+
+.. code-block:: BASH
+
+    npm install
+
+.. vale off
+
+Configure Babel, ESLint, Prettier
+=================================
+
+.. vale on
+
+Use the template files provided. For example .eslintrc.temp
+
+How to test standalone
+**********************
+
+.. code-block:: BASH
+
+    npm run start-helloWorld
+    or
+    npm run start-mautic
+
+In order for ``start-mautic`` to work, a running DDEV container has to be present. 
+
+If you are using another development environment you need to update some paths in ``Demo/mautic/index.html``.
```

Comment:
Let's be kind and give some examples here? You can use code blocks to include code samples.